ds-ingest/config.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+EMBEDDING_MODEL = "text-embedding-ada-002"
+LARGE_EMBEDDING_MODEL = "text-embedding-3-large"
+PUB_LISTING_URL = "https://www.designsafe-ci.org/api/publications/v2"
+NEO4J_URI = os.environ.get("NEO4J_URI", "neo4j://localhost:7687")
```

Comment:
When we do this for real, we're going to want to pass some authentication or other custom config when we connect to Neo4j. Let's create a `get_neo4j_driver()` function that can be customized for either local or remote ingest, and call it in every place we're currently using `GraphDatabase.driver(NEO4J_URI)`